datus/tools/db_tools/sqlite_connector.py

```
@@ -2,74 +2,366 @@
+        self, sql: str, result_format: Literal["csv", "arrow", "pandas", "list"] = "csv"
+    ) -> ExecuteSQLResult:
+        """Execute a SELECT query."""
+        try:
+            self.connect()
+            cursor = self.connection.cursor()
+            cursor.execute(sql)
+            rows = cursor.fetchall()
+            columns = [desc[0] for desc in cursor.description] if cursor.description else []
+
+            # Convert to list of dicts
+            result_list = [dict(zip(columns, row)) for row in rows]
```

Ruff (0.14.3)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)